index.html

```diff
@@ -7,24 +7,34 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="./styles.css" />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet" />
 </head>
 
 <body>
     <nav>
         <h1>IdeaBox</h1>
     </nav>
     <main>
-        <section>
+        <section class="topPane">
             <form>
-                <label for="title">Title:</label>
-                <input type="text" id="title" name="title">
-                <label for="body">Body:</label>
-                <input type="text" id="body" name="body">
-                <button>Save</button>
+                <div class="inputContainer">
+                    <label for="title">Title</label>
+                    <input type="text" id="title" name="title">
+                </div>
+                <div class="inputContainer">
+                    <label for="body" id="bodyLabel">Body</label>
+                    <textarea id="body" name="body" rows="2" cols="40" maxlength="200"></textarea>
+                </div>
+                <div class="inputContainer">
+                    <button>Save</button>
+                </div>
+            
+                
             </form>
         </section>
-        <section></section>
+        <section class="lowerPane">
+
+        </section>
```

Comment:
I don't believe these additional sections are necessary as we already had section tags dividing the top and bottom of the main tag element.